docs/layer_configuration.md

```diff
@@ -0,0 +1,342 @@
+
+#### Example Usage On Windows:
+
+For example, if a Vulkan SDK is installed in `C:\VulkanSDK\1.2.198.0`, execute the following in a Command Window:
+
+```
+C:\> set VK_LAYER_PATH=C:\VulkanSDK\1.2.198.0\Bin
+```
+
+#### Example Usage on Linux
+
+For Linux, if Vulkan SDK 1.2.198.0 was locally installed in `/sdk` and `VULKAN_SDK=/sdk/1.2.198.0/x86_64`:
+
+```
+$ export VK_LAYER_PATH=$VULKAN_SDK/lib/vulkan/layers
+$ export LD_LIBRARY_PATH=$VULKAN_SDK/lib:$VULKAN_SDK/lib/vulkan/layers
+```
+
+#### Example Usage on MacOS
+
+For macOS, if Vulkan SDK 1.2.198.0 was locally installed in `/sdk` and `VULKAN_SDK=/sdk/1.2.198/macOS`:
+
+```
+$ export VK_LAYER_PATH=$VULKAN_SDK/share/vulkan/explicit_layers.d
+$ export DYLD_LIBRARY_PATH=$VULKAN_SDK/lib
+```
+
+### Enabling and ordering Vulkan Layers
+
+Originally, the Vulkan Desktop Loader provided `VK_INSTANCE_LAYERS` to enable layers from the command-line.
+However, starting with the Vulkan Loader built against the 1.3.234 Vulkan headers, the `VK_LOADER_LAYERS_ENABLE` environment
+variable was added to allow for more easily enabling Vulkan layers.
+The newer Loaders will continue to accept the original `VK_INSTANCE_LAYERS` environment variable for some time, but it is
+considered deprecated.
+
+#### Vulkan 1.3.234 Loader and Newer (`VK_LOADER_LAYERS_ENABLE`)
+
+The easiest way to enable a layer with a more recent drop of the Vulkan Loader is using the `VK_LOADER_LAYERS_ENABLE
+environment variable.
+This environment variable accepts a case-insensitive, comma-delimited list of globs which can be used to define
+the layers to load.
+
+For example, previously if you wanted to enable the Api Dump layer and the Validation layer, you would have to set
+`VK_INSTANCE_LAYERS` equal to the full name of each layer:
+
+```
+VK_INSTANCE_LAYERS=VK_LAYER_LUNARG_api_dump;VK_LAYER_KHRONOS_validation
+```
+
+Now, with `VK_LOADER_LAYERS_ENABLE`, you simply can use stars where you don't want to fill in the full name:
+
+```
+VK_LOADER_LAYERS_ENABLE=*api_dump,*validation
+```
+
+##### Example Usage On Windows:
+
+```
+C:\> set VK_LOADER_LAYERS_ENABLE=*api_dump,*validation
+```
+
+##### Example Usage On Linux/macOS:
+
+```
+$ export VK_LOADER_LAYERS_ENABLE=*api_dump,*validation
+```
+
+More info about the new layer filtering environment variables can be found in the `Layer Filtering` section of the
+of the [Loader Layer Documentation](https://github.com/KhronosGroup/Vulkan-Loader/blob/main/docs/LoaderLayerInterface.md).
+
+#### Older Loaders
+
+Older Vulkan Desktop loaders will not accept the filtering environment variable, and so must continue using the original
+`VK_INSTANCE_LAYERS` environment variable.
+
+
+##### Example Usage On Windows:
+
+The variable should include a semicolon-separated list of layer names to activate.
+Note that order is relevant, with the initial layer being the closest to the application, and the final layer being closest to the driver.
```

Comment:
The relevance of order is mentioned only for the older loader configuration. If that's also true for a new loader the suggestion to specify this in that section too.

Reply:
Done